account_fiscal_year_closing/models/account_fiscalyear_closing.py

```diff
@@ -0,0 +1,641 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from openerp import _, api, exceptions, fields, models
```

Comment:
Maybe you can replace all `from openerp ...` to `from odoo ...`

Or execute `find . -type f -name '*.py' | xargs sed -i 's/from openerp/from odoo/g'` on root.